styleguide/components/21-atoms/link/_link.scss

```diff
@@ -94,4 +128,21 @@ a {
   &[href^="https://"] {
     @include icon(external-link, after);
   }
+
+  @keyframes bounce {
```

Comment:
Can't we make animations generic? Put them in a generic base/ folder somewhere to reuse later?

Reply:
Yeah, thought of that too.
But these are very, very specific and require pre existing padding etc.
Don't really want to create a 'standalone-link-bounce' animation in a generic folder, that kinda defeats the purpose.